lms/djangoapps/instructor_analytics/tests/test_basic.py

```diff
@@ -531,3 +532,54 @@ def test_invalid_module_state(self):
         datarows = list(student_responses(self.course))
         #Invalid module state response will be skipped, so datarows should be empty
         self.assertEqual(len(datarows), 0)
+
+    def test_problem_with_student_answer_and_answers(self):
+        self.course = get_course(CourseKey.from_string('edX/graded/2012_Fall'))
```

Comment:
course needs to be self as it is used in create_student. I think this class would be better with a setup but that's for the future.